provider/parameter_test.go

```diff
@@ -424,6 +440,54 @@ data "coder_parameter" "region" {
 				require.Equal(t, expected, state.Primary.Attributes[key])
 			}
 		},
+	}, {
+		Name: "NumberValidation_MonotonicWithOptions",
```

Comment:
I used to drop an example [here](https://github.com/coder/terraform-provider-coder/blob/main/examples/resources/coder_parameter/resource.tf) as well.